parm/config/gefs/config.wave

```diff
@@ -30,7 +29,6 @@ case "${waveGRD}" in
     export waveGRDN='1 2 3'
     export waveGRDG='10 20 30'
     export USE_WAV_RMP='YES'
-    export waveMULTIGRID='.true.'
```

Comment:
The entire case "gnh_10m;aoc_9km;gsh_15m") needs to be removed as it is multiple grids.  Simply removing this statement here is insufficient.